jetty9/src/main/docker/jetty_start.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+DIR=$(dirname $0)
+JETTY_START_COMMAND="/scripts/jetty/jetty-start-command"
+
+if [ ! -f $JETTY_START_COMMAND ]; then
+  source $DIR/gen_start_command.sh $JETTY_START_COMMAND
```

Comment:
I'm not convinced that ONBUILD is what we want here after all. An ONBUILD command would be run automatically after the FROM line in a child container's build, so any customization of jetty (i.e. via JETTY_MODULES_ENABLE) in their Dockerfile would not be reflected in the generated start command.

We could in theory generate the start command, either in the ONBUILD (or even in the jetty-runtime container), and then invalidate it if we detect that the user has made config changes that would create a new start command. But I'm not crazy about that idea, since we'd be taking on a good chunk of complexity to be able to detect those changes for marginal gains in container start time, at best. 

IMO, I think it's OK to tell users who are sensitive to container start time to run the `gen_start_command` script at the appropriate place in their dockerfiles.